### ISSUE TITLE
iPad and ScannerViewController orientation

### ISSUE THREAD

**Issue by justarandomdev** [WeTransfer/WeScan#282]
`ScannerViewController` is set to be shown only on portrait mode, which is normal, since otherwise it would have to rotate the camera view port and convert orientation etc.

However, when all orientations are enabled in `Info.plist`, which is required for all apps unless they also require full screen, the `supportedInterfaceOrientations` method/computed property is never called, so it presents this VC in whatever orientation, including landscape, and the view port doesn't show the right orientation.

You can reproduce this by enabling the iPad checkbox in the sample project, and launching the camera in landscape mode.

To remediate this, the only way I've found so far is to specify only a subset of supported orientations on the `Info.plist`, not all of them, and implement the `application(_ application: UIApplication, supportedInterfaceOrientationsFor window: UIWindow?) -> UIInterfaceOrientationMask` method in App Delegate to determine the supported orientations. (This has the side effect of ignoring the missing orientation in the `Info.plist` when showing the launch screen, but then once launched all orientations are supported, based on what the above method returns)

Then `supportedInterfaceOrientations` is properly called, and when modally presenting `ScannerViewController` it presents only in portrait mode, which is what we want to do.
 
However this missing orientation in `Info.plist` triggers a warning in the project (and the build also in App Store Connect) saying it's not authorised.
So to get rid of it, you need to enable "Requires Full Screen" (`UIRequiresFullScreen`) in the `Info.plist` as well, but this has the effect of disabling split view/multitasking on iPad.

To sum up:

- You can reproduce this with the given sample project by enabling iPad support and showing the scanner in landscape mode
- The only way to then show the `ScannerViewController` only on portrait mode is by not including all orientations in the `UISupportedInterfaceOrientations` array in `Info.plist`
- Removing an orientation, triggers then a warning, requiring us to enable `UIRequiresFullScreen `
- Then it works but has the unfortunate effect of disabling multitasking/split view mode

This issue might be linked to #216 

Thanks in advance.

**Comment by willm132** [comment]
Has anyone found a fix to allow this in landscape only, not portrait?
